tests/models/musicgen/test_modeling_musicgen.py

```diff
@@ -1415,170 +1385,10 @@ def test_model_get_set_embeddings(self):
             lm_heads = model.get_output_embeddings()
             self.assertTrue(lm_heads is None or isinstance(lm_heads[0], torch.nn.Linear))
 
-    def _get_input_ids_and_config(self, batch_size=2):
```

Comment:
(all this deleted code corresponds to overwritten functions that no longer need to be overwritten)

Reply:
love it!

Reply:
That's very nice